scripts/testing/xbox-ssl.conf

```diff
@@ -62,31 +58,20 @@
     RewriteRule ^/$ /console/status [R]
 
     # Redirect assets (see assetPrefix).
+    # TODO(burdon): All three of these rules would go away if the prefix were acting as it should.
```

Comment:
If available reference doc in external project for this. e.g. prefix were acting as described in: http://some.doc.site/doc